microbench/pom.xml

```diff
@@ -73,14 +73,19 @@
     <dependency>
       <groupId>org.openjdk.jmh</groupId>
       <artifactId>jmh-core</artifactId>
-      <version>0.9</version>
+      <version>1.7.1</version>
```

Comment:
No this wasn't required ... just an artifact of having some initial trouble getting the benchmarks to run.  Probably not a bad idea to upgrade though.